roles/kvirt_vm/tasks/create-vm.yml

```diff
@@ -0,0 +1,45 @@
+---
+- name: "Validate VM name"
+  assert:
+    that: vm.name is defined
+    fail_msg: VM name is required
```

Comment:
Would it be best to move this to the validation.yml ? 

This may be helpful:
https://github.com/redhatci/ansible-collection-redhatci-ocp/blob/73c7d7f4ddb8f669df10f21bbb1720ed04dd0365/roles/sriov_config/tasks/validation.yml#L32-L36

Reply:
As we create the VMs in the loop it is necessary to validate the VM name and other specific settings on each iteration.  The validation checks generic requirements, things specific for the VM need to be validated in the loop.

This role is not aware of the sriov_config_file

Reply:
Thanks for the clarification, I moved the validation for all items.